Resources/public/css/theme/views/editpreview.css

```diff
@@ -5,34 +5,61 @@
 
 .ez-view-editpreviewview {
     background: #f3f3f3;
+    color: #555;
 }
 
-.ez-preview-content .ez-preview-hide {
-    color: #555;
+.ez-view-editpreviewview .ez-preview-iframe-wrapper:before {
+    transition: opacity ease 0.3s;
+    opacity: 0;
+
```

Comment:
I guess it's for separate the part that will be changed with transition: https://github.com/ezsystems/PlatformUIBundle/pull/366/files#diff-e5d16b0809fbbf1b577e2b9454cadbf7R27